src/style.css

```diff
@@ -2,2 +1,103 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: #f9f9f9;
+  width: 100%;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.main-header {
+  width: 100%;
+  max-width: 1200px;
+  height: 60px;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it to avoid overflow or accidental scroll bar. It seems like the height is fixed for `.main-header`, which might not be necessary unless there's a specific design requirement.